lib/localstorage-memory.js

```diff
@@ -15,7 +15,11 @@
    * @returns {String|null}
    */
   localStorageMemory.getItem = function (key) {
-    return cache[key] || null
+    if (key in cache) {
```

Comment:
I’ve changed your implementation slightly. `if (cache[key]) { }`  still would be false if the value is `''`. But `if (key in cache) {}` works. Your test is great 👌

Reply:
Cool :) Is there anything left would I do?